tests/MongoDBDriverManager_debugInfo.phpt

```diff
@@ -34,7 +34,7 @@ array(2) {
       ["is_passive"]=>
       bool(false)
       ["last_is_master"]=>
-      array(8) {
+      array(9) {
```

Comment:
I did that for the PHPC one already - the problem is more that different versions of MongoDB include different items. Hence me doing: https://github.com/mongodb/mongo-php-driver/pull/378/commits/6462468f8eb3b469afab355ca88b2aa760c38500#diff-3bb29f65077b3f49f6e05c7d78fdab99R53

Will fix that here too.